fhirstore/fhirstore.py

```diff
@@ -314,33 +319,45 @@ def search(
                 {"resource": h["_source"], "search": {"mode": "match"}}
                 for h in hits["hits"]["hits"]
             ],
-
             "total": hits["hits"]["total"]["value"],
         }
 
         if elements:
             bundle["tag"] = {"code": "SUBSETTED"}
 
         if include:
-            #For each result instance
+            included_hits = {}
+            # For each result instance
             for item in bundle["items"]:
-                #For each attribute to include
+                # For each attribute to include
                 for attribute in include:
-                    # split the reference attribute "Practioner/123" into a 
+                    # split the reference attribute "Practioner/123" into a
                     # resource "Practioner" and an id "123"
-                    included_resource, included_id = re.split(
-                        "\/", item[attribute]["reference"], maxsplit=1
-                    )
+                    try:
+                        included_resource, included_id = re.split(
+                            "\/", item["resource"][attribute]["reference"], maxsplit=1
+                        )
+                        included_hits = self.es.search(
+                            body={
+                                "query": {
+                                    "simple_query_string": {"query": included_id, "fields": ["id"],}
+                                }
+                            },
+                            index=f"fhirstore.{included_resource.lower()}",
+                        )
+                    except KeyError as e:
+                        logging.warning(f"Attribute: {e} is empty")
```

Comment:
Could you provide an example of warning printed? I'm not sure if the warning is describes very well the situation

Reply:
If I do Patient?_include=Patient:generalPractitioner, I get something like : "WARNING:root:Attribute: 'generalPractitioner' is empty"

Reply:
Wouldn't it be important to also have the information of the whole query?
I don't know if it's east to do right now, if not I think it would be thinking about it when refactoring the search

Reply:
Also, the problem I had here is that I'm not sure where the KeyError happens in the try

Reply:
It happens with the split, if there is nothing at item["attribute"]["reference"]. It should not prevent the search from being conducted that's why I put it as a logging warning instead of an operation outcome. What do you mean by "have the information of the whole query"?